tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -605,6 +619,19 @@ def test_tf_estimator_autolog_logs_metrics(tf_estimator_random_data_run):
     assert all((x.step - 1) % 100 == 0 for x in metrics)
 
 
+def test_tf_estimator_v1_autolog_logs_metrics(tmpdir):
```

Comment:
Can we mark this test as `large`? Otherwise, not sure it's being run right now. `@pytest.mark.large`